plotting/utils.py

```diff
@@ -164,11 +163,20 @@ def _map_plot(points, grid_loc, path=True, quiver=True):
         zorder=1,
     )
 
-    img = plt.imread(current_app.config["SHAPE_FILE_DIR"] + "/cartopy_resources/bluemarble.png")
-    img_extent = (-180, 180, -90, 90)
-    m.imshow(
-        img, origin="upper", extent=img_extent, transform=ccrs.PlateCarree(), zorder=1
-    )
+    try:
+        img = plt.imread(
+            current_app.config["SHAPE_FILE_DIR"] + "/cartopy_resources/bluemarble.png"
+        )
+        m.imshow(
+            img,
+            origin="upper",
+            extent=(-180, 180, -90, 90),
+            transform=ccrs.PlateCarree(),
+            zorder=1,
+        )
+    except FileNotFoundError:
+        m.add_feature(cfeature.LAND)
+        m.add_feature(cfeature.OCEAN)
```

Comment:
nit: could you please add a basic `print` statement here so we can capture exception case in our rudimentary logs?